ocaml/xapi/storage_mux.ml

```diff
@@ -18,6 +18,19 @@ module D = Debug.Make (struct let name = "mux" end)
 
 open D
 
+let with_tracing ~name ~dbg f =
+  let dbg, parent = Context.tracing_of_dbg dbg in
+  let tracer = Tracing.get_tracer ~name in
```

Comment:
`name` here is the name of the operation; don't we need the name of the tracer here?

Reply:
the `Tracer` name is not the `TracerProvider` name, its somewhat redundent currently but a tracer having a name is in the spec.